guides/v2.2/payments-integrations/base-integration/payment-action.md

```diff
@@ -25,7 +25,7 @@ Configure the command as described in [Gateway Command]({{ page.baseurl }}/payme
 
 Configuring the gateway command and adding it to command pool (`app/code/Magento/Braintree/etc/di.xml`):
 
-```xml
+xml
```

Comment:
```suggestion
```xml
```
Why you delete the block code? Maybe I missed something.

Reply:
@osrecio 
These changes come because of master merge in this PR.
I have updated changes.

Reply:
Thanks @magesanjay . Check above comment. Of differences between versions.

Reply:
@osrecio 
I have updated as per above comment.